unitary/alpha/qubit_effects_test.py

```diff
@@ -30,6 +30,33 @@ def test_flip(simulator, compile_to_qubits):
     alpha.Flip()(piece)
     assert board.circuit == cirq.Circuit(cirq.X(cirq.NamedQubit("t")))
 
+@pytest.mark.parametrize("compile_to_qubits", [False, True])
+@pytest.mark.parametrize("simulator", [cirq.Simulator, SparseSimulator])
+def test_partial_flip(simulator, compile_to_qubits):
+    board = alpha.QuantumWorld(sampler=simulator(), compile_to_qubits=compile_to_qubits)
+    piece = alpha.QuantumObject("t", 0)
+    board.add_object(piece)
+    alpha.Flip(effect_fraction=0.25)(piece)
+    assert board.circuit == cirq.Circuit(cirq.X(cirq.NamedQubit("t")) ** 0.25)
+
```

Comment:
Add a test to verify the cumulative effect of multiple fractional ops is as expected?